neurow/lib/neurow/observability/observability.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Neurow.Observability do
+  use Prometheus.Metric
+
+  def setup() do
```

Comment:
**Suggestion** 🤔 : can it be useful to create a collection of modules for observability and do something like this

```elixir
# Define a list of modules to setup
observability_modules = [
  Neurow.Observability.MessageBrokerStats,
  Neurow.Observability.HttpInterfacesStats,
  Neurow.Observability.MetricsPlugExporter,
  Neurow.Observability.SecurityStats,
  Neurow.Observability.SystemStats
]

# Iterate over the list and call the setup function on each module
Enum.each(observability_modules, fn module -> module.setup() end)
```

Reply:
I think, for now it is easer to read without indirection.